.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
+  rubocop:
+    display_cop_names: true
```

Comment:
#### `REQUIRED`

#### Add appropriate linter files to this branch.
  > [Set up a code linter](https://github.com/microverseinc/linters-config/tree/master/ruby).

Use [this file](https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml) instead. :point_down: 

```.yml
linters:
  rubocop:
    display_cop_names: true
files:
  ignore:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
```

### Also, add the `.rubocop.yml` file. This can be found [here](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml).